Management/src/AllActuators/Actuators/manifest.yml

```diff
@@ -0,0 +1,7 @@
+---
+applications:
+- name: sbo-integration
+  memory: 1024M
+  command: cd ${DEPS_DIR}/0/dotnet_publish && exec ./Steeltoe.Actuators --server.urls https://sbo-integration.apps.pcfone.io
```

Comment:
Won't the buildpack will set this automatically?
```suggestion
```

Reply:
It should... I think this is leftover from some troubleshooting. I'll remove it and do a push to make sure, then update this file.

Reply:
This was left over when troubleshooting why the wrong .NET version (3.1) was being targeted by default. The `buildpack.yml` file resolved that issue. This has been removed.